docs/apis/application.yaml

```diff
@@ -0,0 +1,194 @@
+            $ref: '#/definitions/model.Application'
+        "400":
+          description: Bad Request
+          schema:
+            type: string
+        "500":
+          description: Internal Server Error
+          schema:
+            type: string
+      summary: Create an application
+      tags:
+      - applications
+  /applications/{id}:
+    delete:
+      consumes:
+      - application/json
+      description: Delete an application using its ID
+      parameters:
+      - description: Application ID
+        in: path
+        name: id
+        required: true
+        type: string
+      produces:
+      - application/json
+      responses:
+        "204":
+          description: No Content
+          schema:
+            type: string
+        "400":
+          description: Bad Request
+          schema:
+            type: string
+        "404":
+          description: Not Found
+          schema:
+            type: string
+        "500":
+          description: Internal Server Error
+          schema:
+            type: string
+      summary: Delete an application
+      tags:
+      - applications
+    get:
+      consumes:
+      - application/json
+      description: Retrieve a specific application using its ID
+      parameters:
+      - description: Application ID
+        in: path
+        name: id
+        required: true
+        type: string
+      produces:
+      - application/json
+      responses:
+        "200":
+          description: OK
+          schema:
+            $ref: '#/definitions/model.Application'
+        "400":
+          description: Bad Request
+          schema:
+            type: string
+        "404":
+          description: Not Found
+          schema:
+            type: string
+        "500":
+          description: Internal Server Error
+          schema:
+            type: string
+      summary: Get an application by ID
+      tags:
+      - applications
+    put:
+      consumes:
+      - application/json
+      description: Update the details of an existing application
+      parameters:
+      - description: Application ID
+        in: path
+        name: id
+        required: true
+        type: string
+      - description: Updated application data
+        in: body
+        name: application
+        required: true
+        schema:
+          $ref: '#/definitions/model.Application'
+      produces:
+      - application/json
+      responses:
+        "200":
+          description: OK
+          schema:
+            $ref: '#/definitions/model.Application'
+        "400":
+          description: Bad Request
+          schema:
+            type: string
+        "404":
+          description: Not Found
+          schema:
+            type: string
+        "500":
+          description: Internal Server Error
+          schema:
+            type: string
+      summary: Update an application
+      tags:
+      - applications
+securityDefinitions:
+  BasicAuth:
+    type: basic
+swagger: "2.0"
```

Comment:
can this be v3 ?

Reply:
as i've seen `swag` doesn't generate v3 specs